Sources/Plot/API/HTML.swift

```diff
@@ -117,6 +119,9 @@ public protocol HTMLImageContainerContext: HTMLContext {}
 /// Context shared among all HTML elements that act as some form
 /// of link to an external resource, such as `<link>` or `<a>`.
 public protocol HTMLLinkableContext: HTMLContext {}
+/// Context shared among all HTML elements that support the `data` (URL)
+/// attribute, such as `<object>.`
+public protocol HTMLLinkableDataContext: HTMLContext {}
```

Comment:
Since it looks like only `<object>` is using this protocol, is it really needed? It seems like we can just use `ObjectContext` as the constraint for the new `data` methods, right?

Reply:
Absolutely. After doing some research, I found no other HTML element that has a `data` attribute. Consider it fixed.